src/components/dashboard/widget/dashboard-stack-mode.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum DashboardStackMode {
+    Regular,
+    Stacked,
+    Auto // Determine the mode automatically based on dashboard width
```

Comment:
If you add this comment above `Auto` using a documentation comment it will show on tooltips in code which might be helpful, e.g.:

```
/** Determine the mode automatically based on dashboard width. */
Auto
```